src/components/Widget/IntegrationsWidget.tsx

```diff
@@ -55,6 +55,7 @@ const IntegrationsWidget: FunctionComponent = () => {
   const isPagerDutyEnabled = useFlag('platform.integrations.pager-duty');
   const integrationsData = createIntegrationsData(isPagerDutyEnabled, hasSourcesPermissions, hasIntegrationsPermissions);
 
+  // eslint-disable-next-line prettier/prettier
```

Comment:
No need to disable prettier.
```suggestion
```

Reply:
I only added because I kept getting a prettier error telling me to delete `<number, boolean>` from the line below